packages/react-scripts/scripts/start-ssr.js

```diff
@@ -0,0 +1,239 @@
+      paths.publicUrlOrPath.slice(0, -1)
+    );
+
+    // @brs-begin
+    const webUi = new WebCompilerUi(
+      'web',
+      { color: 'bgBlue' },
+      { appName, urls, useYarn }
+    );
+    ui.register(webUi);
+
+    const compiler = createCustomCompiler(webUi, {
+      appName,
+      config,
+      devSocket,
+      urls,
+      useYarn,
+      useTypeScript,
+      tscCompileOnError,
+      webpack,
+    });
+
+    statusFile.init(compiler, paths.appBuildWeb);
+    // @brs-end
+
+    // Load proxy config
+    const proxySetting = require(paths.appPackageJson).proxy;
+    const proxyConfig = prepareProxy(
+      proxySetting,
+      paths.appPublic,
+      paths.publicUrlOrPath
+    );
+    // Serve webpack assets generated by the compiler over a web server.
+    const serverConfig = {
+      ...createDevServerConfig(proxyConfig, urls.lanUrlForConfig),
+      host: HOST,
+      port,
+    };
+
+    // @brs-begin
+    serverConfig.devMiddleware.writeToDisk = filePath => {
+      return /loadable-stats\.json/.test(filePath);
+    };
+    // @brs-end
+
+    const devServer = new WebpackDevServer(serverConfig, compiler);
+    // Launch WebpackDevServer.
+    devServer.startCallback(() => {
+      if (isInteractive) {
+        clearConsole();
+      }
+
+      if (env.raw.FAST_REFRESH && semver.lt(react.version, '16.10.0')) {
+        console.log(
+          chalk.yellow(
+            `Fast Refresh requires React 16.10 or higher. You are using React ${react.version}.`
+          )
+        );
+      }
+
+      console.log(chalk.cyan('Starting the development server...\n'));
+      openBrowser(urls.localUrlForBrowser);
+
+      // @brs-begin
+      // This method will clear the console, so we put it after CRA has logged the messages above.
+      ui.start();
+      // @brs-end
+    });
+
+    // @brs-begin
+    const { fork } = require('child_process');
+    const { join } = require('path');
+
+    // We do this in a background process for performance - multicore, yo.
+    const ssr = fork(join(__dirname, '../backpack-addons/ssr/forkSsr'));
+
+    const ssrUi = new ProcessMessageCompilerUi(ssr, 'ssr', {
+      color: 'bgMagenta',
+    });
+    ui.register(ssrUi);
+
+    ssr.on('exit', code => {
+      ssrUi.log(chalk.red(`Fatal! Process ended with code ${code}`));
+      devServer.close();
+      process.exit();
+    });
+    // @brs-end
+
+    ['SIGINT', 'SIGTERM'].forEach(function (sig) {
+      process.on(sig, function () {
+        devServer.close();
+        // @brs-begin
+        ssr.kill();
+        process.exit();
+        // @brs-end
+      });
+    });
+
+    if (process.env.CI !== 'true') {
+      // Gracefully exit when stdin ends
+      process.stdin.on('end', function () {
+        devServer.close();
+        // @brs-begin
+        ssr.kill();
+        process.exit();
+        // @brs-end
+      });
+    }
+  })
+  .catch(err => {
+    if (err && err.message) {
+      console.log(err.message);
+    }
+    process.exit(1);
+  });
```

Comment:
Ensure this file is similar to start.js of CRA v5.

Reply:
This file copy from start.js, and then update ssr config.